Formula/ttygif.rb

```diff
@@ -0,0 +1,18 @@
+class Ttygif < Formula
+  desc "Converts a ttyrec file into gif files"
+  homepage "https://github.com/icholy/ttygif"
+  url "https://github.com/icholy/ttygif/archive/1.4.0.tar.gz"
+  sha256 "6ca3dc5dcade2bdcf8000068ae991eac518204960c157634d92f87248c3cee2a"
+
+  depends_on "imagemagick"
+  depends_on "ttyrec"
+
+  def install
+    system "make"
+    bin.install "ttygif"
```

Comment:
It's already there: https://github.com/icholy/ttygif/blob/master/Makefile#L19

How do I use that instead?